### ISSUE TITLE
calendar_registry.py imports

### ISSUE THREAD

**Issue by TheAIArchitect** [rsheftel/pandas_market_calendars#86]
It looks like when you optimized imports in [this commit](https://github.com/rsheftel/pandas_market_calendars/commit/2d39d7599208512cdd0d83aedd20fc270639291a#diff-0770b920f3d4639d1bf233098c8c092e), you removed the imports added in [this commit](https://github.com/rsheftel/pandas_market_calendars/commit/7b166acbb3fc122929f38441b294bfc7c4f39786), which re-created the empty registry issue that you had solved in the first commit.

This produces an error similar to:

> 'Class {} is not one of the registered classes: {}'.format(name, cls._regmeta_class_registry.keys()))
> RuntimeError: Class NYSEExchangeCalendar is not one of the registered classes: dict_keys(['MarketCalendar'])

(Added error message that so people can search and find this by the error if they encounter it.)

**Comment by rsheftel** [comment]
Can you provide an example for when this error occurs? I do not see it in any of the tests or in my use of the package.

**Comment by rmorel** [comment]
I have the same Issue. The code that I tested is:

```
import pandas_market_calendars as mcal
nyse = mcal.get_calendar('NYSE')
```

That also gives me 

> 
> ~\Anaconda3\envs\economy\lib\site-packages\pandas_market_calendars\class_registry.py in _regmeta_class_factory(cls, name)
>       9     else:
>      10         raise RuntimeError(
> ---> 11             'Class {} is not one of the registered classes: {}'.format(name, cls._regmeta_class_registry.keys()))
>      12 
>      13 
> 
> RuntimeError: Class NYSE is not one of the registered classes: dict_keys(['MarketCalendar']) 

Also, `mcal.get_calendar_names()` returns a empty list.

I installed the package via pip and `mcal.__version__` returns version 1.3

**Comment by TheAIArchitect** [comment]
I think that if you explicitly import the calendar you want it'll work, but I just re-added the imports to calendar_registry.py

**Comment by rsheftel** [comment]
@TheAIArchitect can you do a PR of what you did to solve the problem

**Comment by rmorel** [comment]
A update on the Issue.

I use the [Anaconda](https://www.anaconda.com/) distribution. I updated my packages just now and now the code works again.

The package version is the same, `mcal.__version__` returns version 1.3. And now `mcal.get_calendar_names()` returns

> ['BMF',
>  'CFE',
>  'CME',
>  'CBOT',
>  'COMEX',
>  'NYMEX',
>  'EUREX',
>  'ICE',
>  'ICEUS',
>  'NYFE',
>  'JPX',
>  'LSE',
>  'NYSE',
>  'stock',
>  'NASDAQ',
>  'BATS',
>  'OSE',
>  'SIX',
>  'TSX',
>  'TSXV',
>  'SSE',
>  'HKEX']

I can't reproduce the problem anymore.

@TheAIArchitect which python distribution are you using? Have you tried updating your packages to see if the problem persists?

**Comment by TheAIArchitect** [comment]
I just put a zip of the master branch of the project into a bigger project and tried to run it using Python 3.6.7.

I don't have a clone of this project, but the only change I made was to paste back the import lines added in the second commit I linked (https://github.com/rsheftel/pandas_market_calendars/commit/7b166acbb3fc122929f38441b294bfc7c4f39786).

**Comment by rsheftel** [comment]
Including those imports cause a circular import and causes all tests to fail. Need to figure this out.

**Comment by rsheftel** [comment]
This is fixed in c033ab8f85d262bcdcb383e0c214dc80ff3ba6c1 and pushed to PyPi as v1.3.1
